src/com/content-item/item.js

```diff
@@ -66,7 +66,7 @@ export default class ContentItem extends Component {
 		});
 		for (let i = 0; i < 5; i ++) {
 			this.state.linkNames[i] = node.meta['link-0' + (i+1) + '-name'];
-			this.state.linkUrls[i] = node.meta['link-0' + (i+1) + '-name'];
+			this.state.linkUrls[i] = node.meta['link-0' + (i+1) + '-url'];
```

Comment:
Don't use `-url`. Instead, have no suffix. Assume that `link-01` is a URL, and if no target (`B`) is set on the link, that `link-01-name` is how to set a name. When `B` is non-zero, the Tag node's `name` is used instead.